src/plugins/intel_gpu/src/kernel_selector/kernels/gemm/gemm_kernel_tiled_opt.cpp

```diff
@@ -261,6 +261,10 @@ bool GemmKernelTiledOpt::Validate(const Params& params, const optional_params& o
 
     for (size_t input_idx = 0; input_idx < gmm_params.inputs.size(); ++input_idx) {
         auto& input = gmm_params.inputs[input_idx];
+        if (!Tensor::SimpleLayout(input.GetLayout())) {
+            return false;
+        }
```

Comment:
As mentioned in the meeting, proper fix might be to add reorder because of potential performance issue.
* Please check the performance impact of this change
* Please review alternative approach to add reorder.

Reply:
I tested 2nd approach which is adding reorder before issued Gemm layer for target condition. The performance on the target model has no meaningful diff(<0.1 FPS). After team internal discussion, I decided to use current approach.

Reply:
Could you share more detail of your experiment? Which gemm is chosen if this tiled_opt is not chosen? How much is the performance difference for this layer itself?

Reply:
gemm selects ref kernel by using current PR.
This PR(gemm:ref krenel): 32.39 FPS
Adding reorder(gemm:tiled_opt kernel): 32.46 FPS